node/src/BaseClient.ts

```diff
@@ -3416,6 +3417,33 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the positions (longitude, latitude) of all the specified `members` of the
+     * geospatial index represented by the sorted set at `key`.
+     *
+     * See https://valkey.io/commands/geopos for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param members - The members for which to get the positions.
+     * @returns A 2D `array` which represents positions (longitude and latitude) corresponding
+     *     to the given members. If a member does not exist, its position will be `null`.
+     *
+     * @example
+     * ```typescript
+     * await client.geoadd("mySortedSet", new Map([["Palermo", new GeospatialData(13.361389, 38.115556)], ["Catania", new GeospatialData(15.087269, 37.502669)]]));
+     * const result = await client.geopos("mySortedSet", ["Palermo", "Catania", "NonExisting"]);
+     * // When added via GEOADD, the geospatial coordinates are converted into a 52 bit geohash, so the coordinates
+     * // returned might not be exactly the same as the input values
+     * console.log(result); // Output: [[13.36138933897018433, 38.11555639549629859], [15.08726745843887329, 37.50266842333162032], null]
+     * ```
+     */
+    public geopos(
```

Comment:
Can it throw an error?

Reply:
Nothing command-special, but it can throw same errors as all other commands do.